lte/gateway/python/integ_tests/s1aptests/test_ipv6_non_nat_dp_ul_tcp.py

```diff
@@ -0,0 +1,142 @@
+"""
```

Comment:
Code formatting seems to be pending, so I assume this PR is in intermediate stage and formatting will be done once this PR changes are finalized.

Reply:
This test is moved to a different PR